src/EditorFeatures/Core/Implementation/Classification/SemanticClassificationUtilities.cs

```diff
@@ -127,5 +131,27 @@ internal static class SemanticClassificationUtilities
                 throw ExceptionUtilities.Unreachable;
             }
         }
+
+        private static void CombineSpans(List<ClassifiedSpan> spans, IEnumerable<ClassifiedSpan> newSpans)
```

Comment:
Given the comment about `ClassificationUtilities.GetOrCreateClassifiedSpanList(...)` above, this method might not be needed.